pimd/pim_str.h

```diff
@@ -26,6 +26,10 @@
 
 #include <prefix.h>
 
+#define PIM_ADDR in_addr
+#define PIM_UN_PFX(p) ((p).u.prefix4)
+#define pim_inet_dump prefix_mcast_inet4_dump
```

Comment:
NAAAAAAK.

This runs counter to any sense of C coding style. Defining a single "word" that isn't even the entire type?!?

`typedef`, please.

And the helpers should be `static inline` functions.

Reply:
Fixed